src/herder/TxSetFrame.cpp

```diff
@@ -398,56 +481,66 @@ TxSetFrame::previousLedgerHash() const
     return mPreviousLedgerHash;
 }
 
-TxSetFrame::Transactions const&
+TxSetFrame::Transactions
```

Comment:
Just a consideration: if we use this frequently we might consider using a visitor instead in order to avoid copying the whole tx set.

Reply:
I ended up changing this function to return const ref to txs in a particular phase - its probably safer to force callers to deal with individual phases instead of pooling all txs together.